.github/workflows/build-and-release-stable.yml

```diff
@@ -59,9 +59,15 @@ jobs:
         with:
           version: latest
           args: release --rm-dist --skip-validate
+      - name: bump homebrew-core formula
+        uses: mislav/bump-homebrew-formula-action@v2
+        with:
+          formula-name: mesheryctl
+          download-url: https://github.com/meshery/meshery.git
+        env:
+          COMMITTER_TOKEN: ${{ secrets.HOMEBREW_COMMITTER_TOKEN }}
```

Comment:
```suggestion
          COMMITTER_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
```

Reply:
Just want to confirm that it has `public_repo` access.